Simple Spec Page

roselynemerald opened the pull request:
Simple spec page with dummy text

Commit:
create simple spec page

rzalog commented on client/src/components/Spec.vue:
```diff
@@ -0,0 +1,96 @@
+  border: 1px;
+  padding: 0 0 15px;
+  margin: 15px 15px;
+ -webkit-box-shadow: 2px 3px 8px #eee;
```
Did we decide to go with this styling (the kind of intense box shadow) for assignments? This is more of a question for @danielskathyd

danielskathyd reviewed (changes requested):
Looks good overall! Some changes:

1. Try adjusting margins so that the box aligns closer to the black border of the navbar
2. Make sure that text doesn't overflow past the edge of the box when condensing the browser
3. Decrease spacing above title and increase spacing above box to more closely match the Figma design
4. After all that is done, make sure the box is still responsive to increasing and decreasing browser width

Commit:
modified margins and spacing

roselynemerald commented:
- Adjusted box margins and padding
- Made sure that text doesn't overflow past the edge of the box 
- Changed spacing to more closely match the Figma design

Commit:
Merge branch 'master' of https://github.com/UCLA-IEEE/project-portal into roselyn/spec

Commit:
Centered spec box
